src/components/line/ItemThermometerStations.vue

```diff
@@ -0,0 +1,159 @@
+import IconBus from '@/components/ui/icons/IconBus.vue'
+import IconParking from '@/components/ui/icons/IconParking.vue'
+import type { LineNumber } from '@/model/lines.model'
+import { getColorLine } from '@/services/color'
+import type { BusNumber } from '@/model/bus.model'
+
+const props = defineProps({
+  index: {
+    type: Number,
+    required: true,
+  },
+  line: {
+    type: Number as PropType<LineNumber>,
+    required: true,
+  },
+  name: {
+    type: String,
+    required: true,
+  },
+  desserte: {
+    type: String,
+    required: true,
+  },
+  li_code: {
+    type: String,
+    required: true,
+  },
+  parking: {
+    type: Boolean,
+    required: false,
+    default: false,
+  },
+  is_last_elem: {
+    type: Boolean,
+    required: true,
+    default: false,
+  },
+})
+const stationActive = ref<Boolean>(false)
+const borderColor = ref(getColorLine('border', props.line, 600))
+
+let desserteSplit = props.desserte
+  .split(' ')
+  .filter((bus) => bus !== undefined && bus != '')
+const desserteTab = ref<BusNumber[]>(
+  desserteSplit.map((bus) => {
+    let new_bus: BusNumber = bus as BusNumber
+    return new_bus
+  })
+)
+
+var lineTab = ref<LineNumber[]>([])
+if (props.li_code != '') {
+  let li_code_split = props.li_code.split(' ')
+  lineTab.value = li_code_split.map((line) => {
+    let new_line: LineNumber = parseInt(line) as LineNumber
+    return new_line
+  })
+}
+
+function getClassCircle() {
+  let marginLeftNegative = '-ml-1'
+  let border2px = 'border-[2px]'
+  if (stationActive.value) {
+    return [
+      'min-w-[16px]',
+      'w-[16px]',
+      'h-[16px]',
+      'border-black',
+      marginLeftNegative,
+      border2px,
+    ]
+  }
+  if (props.index == 1 || props.is_last_elem) {
+    return [
+      'min-w-[15px]',
+      'w-[15px]',
+      'h-[15px]',
+      marginLeftNegative,
+      borderColor.value,
+      'border-[4px]',
+    ]
+  }
+  return ['min-w-[8px]', 'w-[8px]', 'h-[8px]', borderColor.value, border2px]
+}
+
+function getClassBeforeCircle() {
+  if (props.index == 1) {
+    return []
+  }
+  var classBefore = [
+    'before:absolute',
+    "before:content-['']",
+    'before:border-l-[2px]',
+    'before:' + borderColor.value,
+  ]
+  let ml = 'before:ml-[1px]'
+  let bottom = 'before:bottom-[65%]'
+  let height = 'before:h-[95%]'
+  if (stationActive.value) {
+    ml = 'before:ml-[5px]'
+    bottom = 'before:bottom-[75%]'
+    height = 'before:h-[70%]'
+  } else if (props.is_last_elem) {
+    ml = 'before:ml-[3px]'
+    height = 'before:h-[85%]'
+  }
+  return classBefore.concat([ml, bottom, height])
+}
+
+const classCircle = computed(() => {
+  var classObject = ['rounded-[50%]', 'mr-2', 'mb-3', 'mt-2']
+  classObject = classObject.concat(getClassCircle())
+  classObject = classObject.concat(getClassBeforeCircle())
+  return classObject
+})
+</script>
+
+<template>
+  <li
+    class="flex items-center mb-1 cursor-pointer"
+    :class="stationActive ? 'bg-[#F1F5F9]' : ''"
```

Comment:
does the color have a Tailwind class? In general, we try to use the style from Tailwind. The designer also choose the color from tailwind, usually they put the tailwind class on the color notes in Figma